main/autogen/autogen.h

```diff
@@ -83,13 +83,18 @@ struct ParsedFile {
     std::string toString(core::Context ctx);
     std::string toMsgpack(core::Context ctx, int version);
     void classlist(core::Context ctx, std::vector<std::string> &out);
-    void subclasses(core::Context ctx, std::map<std::string, std::set<std::string>> &out);
+    void subclasses(core::Context ctx, std::vector<std::string> &absolutePathsToIgnore,
+                    std::vector<std::string> &relativePathsToIgnore,
```

Comment:
could this argument be `const`?

Reply:
[yep](https://github.com/sorbet/sorbet/pull/1040/files#diff-670beace6f2de3dc49c6286fb9310f27R90)